src/Microsoft.Data.SqlClient/src/Microsoft/Data/SqlClient/SqlException.cs

```diff
@@ -99,6 +109,24 @@ public override void GetObjectData(SerializationInfo si, StreamingContext contex
         /// <include file='../../../../../../doc/snippets/Microsoft.Data.SqlClient/SqlException.xml' path='docs/members[@name="SqlException"]/Source/*' />
         override public string Source => TdsEnums.SQL_PROVIDER_NAME;
 
+#pragma warning disable CS1591 // Missing XML comment for publicly visible type or member
```

Comment:
Please make sure all these disabled warning tags are removed and docs are added.